test/integ-new/main_new_test.go

```diff
@@ -180,6 +182,53 @@ func TestAPIAndPublicKeyService(t *testing.T) {
 	}
 }
 
+func TestUIService(t *testing.T) {
+	t.Parallel()
+
+	cases := []struct {
+		name           string
+		wantStatusCode int
+	}{
+		{
+			name:           "test_health_checkpoint",
+			wantStatusCode: 200,
```

Comment:
instead of hard coding, please use http.StatusOK

Reply:
Updated.